app/src/main/res/layout/content_event.xml

```diff
@@ -145,10 +175,44 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:lines="3"
+                android:textSize="@dimen/text_size_small"
+                android:textStyle="bold"
+                android:textColor="@color/black"
+                android:layout_marginLeft="@dimen/layout_margin_large"
+                android:layout_marginRight="@dimen/layout_margin_large"
                 android:text="Description" />
+            <View
+                android:layout_width="match_parent"
+                android:layout_height="0.8dp"
+                android:layout_marginBottom="20dp"
+                android:layout_marginTop="12dp"
+                android:background="@color/black" />
 
-        </LinearLayout>
-    </LinearLayout>
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="8dp"
+                android:layout_marginLeft="@dimen/layout_margin_large"
+                android:layout_marginRight="@dimen/layout_margin_large"
+                android:text="@string/tickets_available"
+                android:textSize="20dp" />
 
+            <ProgressBar
```

Comment:
in this case recycler will appear when tickets are loading (event details may have loaded by then) can we design a better logic to show progress bar only at ticket horizontal recycler or something?
No problem with this too just a enhancement

Reply:
Sure